src/melee/ft/chara/ftPikachu/ftpikachu4.c

```diff
@@ -280,7 +280,10 @@ void ftPikachu_80126614(HSD_GObj* fighter_gobj)
 
     ftPikachuAttributes* pika_attr = fp->x2D4_specialAttributes;
 
-    if (!func_80082888(fighter_gobj, &pika_attr->height_attributes)) {
+    /// @todo Eliminate cast (by changing type of field)
+    if (!func_80082888(fighter_gobj,
+                       (ftCollisionBox*) &pika_attr->height_attributes))
```

Comment:
These are either `f32[6]` or `ftCollisionBox*` depending on where you are in the codebase.